infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -639,6 +639,7 @@ sites:
     name: "Roskilde Bibliotekerne"
     description: "The library site for Roskilde"
     deploy_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIPfCXTAe50+d/RUIu8JSctoCSzfJoEMtZ2OY0AUQdZgo"
+    plan: webmaster
```

Comment:
Do they need a separate entry for the release to use on the module test site?

Reply:
that has not yet been merged in